dialogue/Story/Intro/IntroViolastro3.json

```diff
@@ -10,12 +10,16 @@
     "es": [
       "¡<w>Bwejejeje!</w>"
     ],
+    "pl": [
+      "<w>Bwahahaha!</w>"
```

Comment:
Keep in mind this line is shown alongside a voice line in the game - for consistency with the voice line, use "Błehehehe".

Reply:
Agreed, I think it'd be best to stick to a word that, when pronounced, sounds similar to the voice line in game.

Reply:
Alright, gotta change that